board/piksiv3/rootfs-overlay/etc/init.d/S18cfg

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# initialize config data
+#
+
+case "$1" in
+  start)
+  mkdir -p /cfg
+  SBP_SENDER_ID_HEX=`cat /factory/uuid | tail -c 4`
+  printf "%u" "0x$SBP_SENDER_ID_HEX" > /cfg/sbp_sender_id
```

Comment:
>Variable files—files whose content is expected to continually change during normal operation of the system—such as logs, spool files, and temporary e-mail files.

I'm not convinced `/var` is the place. Maybe we should consolidate `/factory`, `/img_tbl`, and `/cfg` under `/piksi` or similar.